docs/UserGuide.md

```diff
@@ -73,19 +73,39 @@ This is the list of all prefixes used in RHRH, as well as their corresponding pa
-| `lp/`  | Loyalty Point       | A non-negative number that should not exceed 2^31 - 1
+|  `t/`  | Tag (Optional)      | Contains alphanumeric characters. <br>One entity can have multiple tags
+| `lp/`  | Loyalty Point       | A non-negative number that should not exceed 100000
 | `alg/` | Allergy (Optional)  | Similar to `t/`
 | `sr/`  | Special Request (Optional) | Similar to `t/`
-|  `l/`  | Leaves              | Leaves should only contain numbers
+|  `l/`  | Leaves              | Leaves should only contain non-negative numbers that are less than or equals to 365
 | `jt/`  | Job Title           | Job Title should only contain alphanumeric characters and spaces, and should not be left blank
-| `sal/` | Salary              | Salary should contain numbers and should be at least 3 numbers long
-| `sh/`  | Shift (Optional)    | Format: `yyyy-MM-dd HHmm`, e.g. `2021-12-08 0800`. One entity can have multiple shifts <br> When editing shifts, the existing values of shifts will be replaced, i.e editing of these fields are not cumulative <br> If you want to remove all shifts from an entity, you can use `editE INDEX sh/` without specifying any shifts after it <br> **NOTE**: Shifts CAN be in the past, present or future. Flexibility is given to the restaurant on how they want to use it 
+| `sal/` | Salary              | Salary should be numerical and is more than or equals to 100 and less than or equals to 10 million
+| `sh/`  | Shift (Optional)    | One entity can have multiple shifts <br> **NOTE**: Shifts CAN be in the past, present or future. Flexibility is given to the restaurant on how they want to use it <br> Refer [here](#date-time-formatting) for more information on how to format shifts
 | `st/`  | Supply Type         | Supply types should only contain alphanumeric characters and spaces, and it should not be blank
-| `dd/`  | Delivery Details    | Refer [here](#adding-a-supplier-adds) for more details
-| `at/`  | Reserving Date Time | Format: `yyyy-MM-dd HHmm`, e.g. `2021-12-24 2000` |
+| `dd/`  | Delivery Details    | Refer [here](#date-time-formatting) for more details
+| `at/`  | Reserving Date Time | Refer [here](#date-time-formatting) for more details
 |  `r/`  | Remark (Optional)   | Contains alphanumeric characters<br> If you want to remove the remark, you can use edit command with `r/`, without specifying any remark after it
 
+<div markdown="block" class="alert alert-warning">
+
+:information_source: **Notes:**<br>
+
+### Date Time Formatting
+
+* All fields that require both `Date` and `Time` in RHRH are accepted if they follow the format below, where the order of date and time are interchangeable.
+However, time strictly follows the `24 hours` format. 
+
+The following table shows the acceptable formats and relevant examples for `DeliveryDetails`:
+
+| Acceptable Formats | Examples |
+| ----------- | ----------- |
+| yyyy-MM-dd HH:mm | 2021-09-19 13:00 |
+| dd-MM-yyyy HH:mm | 19-09-2021 13:00 |
+| HH:mm yyyy-MM-dd | 14:00 2021-11-10 |
```

Comment:
Also, we mentioned we wanted to either caps or lower case everything right since users wont know the formatting syntax used in `DateTimeFormatter` anyways

Reply:
Yeah wil change all to small